Cargo.toml

```diff
@@ -24,6 +24,7 @@ rustc-serialize = "^0.3.15"
 num = "^0.1.25"
 lazy_static = "^0.1.14"
 quick-error = "^0.1.3"
+pulldown-cmark={ git="https://github.com/google/pulldown-cmark.git"}
```

Comment:
Yes, it is a great project but unfortunately it is not very active.. :/

As for specifying the commit, it should be possible, but I have never done it.
Something like below?

```
pulldown-cmark={ git="https://github.com/google/pulldown-cmark.git", rev="e7c3918 " }
```